packages/platform-ios/src/commands/runIOS/parseIOSDevicesList.ts

```diff
@@ -26,7 +26,7 @@ function parseIOSDevicesList(text: string): Array<Device> {
 
   text.split('\n').forEach((line) => {
     const device = line.match(
-      /(.*?) (\(([0-9\.]+)\) )?\[([0-9A-F-]+)\]( \(Simulator\))?/i,
+      /(.*?) (\(([0-9.]+)\) )?\[([0-9A-F-]+)\]( \(Simulator\))?/i,
```

Comment:
Is this change intentional?

Reply:
no, my editor probably removed that on save